utils/build/docker/dotnet/weblog/Controllers/IastController.cs

```diff
@@ -730,5 +730,119 @@ private MailMessage BuildMailMessage(string name)
+            static byte[] GetSHA1_6() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_7() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_8() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_9() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_10() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_11() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_12() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_13() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_14() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_15() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+        }        
+        
+        [HttpPost("sampling-by-route-method-count/{count}")]
+        public IActionResult TestSecureEmailHtmlInjection_Post([FromForm] int count)
+        {
+            GetSHA1_1();
+            GetSHA1_2();
+            GetSHA1_3();
+            GetSHA1_4();
+            GetSHA1_5();
+            GetSHA1_6();
+            GetSHA1_7();
+            GetSHA1_8();
+            GetSHA1_9();
+            GetSHA1_10();
+            GetSHA1_11();
+            GetSHA1_12();
+            GetSHA1_13();
+            GetSHA1_14();
+            GetSHA1_15();
+
+            return Content("OK");
+
+            static byte[] GetSHA1_1() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_2() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_3() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_4() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_5() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_6() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_7() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_8() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_9() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_10() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_11() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_12() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_13() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_14() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }
+            static byte[] GetSHA1_15() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); }        }

```

Comment:
I guess that the `}` at the end of the line should be really in the next line.
```suggestion
            static byte[] GetSHA1_15() { return SHA1.Create().ComputeHash([0, 1, 2, 3]); })
        }
```